coralproject/talk
# Paginate flagged users and more

## What does this PR do?
- Implement `UserConnection` on the graph
- Allow property `statuses` for filtering in a user query.
- Refactor `FlaggedUsers` related code
- Refactor Community styles
- Add transition effect
- Implement graphql mutation updators.
- Always refetch when switching to flagged Users.

## How to test this PR?
- Have more than 10! flagged users to see pagination ^.^
- See transition effect when removing a user
- See refetch taking place whenever switching to flagged user view.
